packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/lambda/run_test.py

```diff
@@ -0,0 +1,6 @@
+import unittest
```

Comment:
are these tests run automatically when users type `npm run test`? if not is there a way of incorporating it?

Reply:
No they aren't run when doing `npm run test`. But I'm not sure if that's possible, I'll look into it!

Reply:
Working on this, will update the PR soon!

Reply:
Added a new `test/queue.lambda.test.ts` file that runs the Lambda tests along with other unit tests when calling `npm run build`. This file in turn calls a script `test/queue-handler/test.sh` that runs the tests.